### PR TITLE
Refactor category endpoints

### PR DESCRIPTION
**Pull request by shahnami**
- Removes create and delete endpoints
- Adjusts example

### CODE REVIEW

File: packages/sentinel/src/models/category.ts [hunk 1]
```diff
@@ -5,21 +5,37 @@ export interface SaveNotificationCategoryRequest {
   /**
    * @maxItems 2
    */
-  notificationIds: [] | [string] | [string, string];
+  notificationIds: [] | [NotificationReference] | [NotificationReference, NotificationReference];
   stackResourceId?: string;
 }
 
 // copied from /models/src/types/notification-category.res.d.ts
+export type NotificationType =
```
Comment (BjMrq) [inline packages/sentinel/src/models/category.ts:13]: I think this already exist in `packages/sentinel/src/models/notification.ts`

File: packages/sentinel/src/models/category.ts [hunk 2]
```diff
@@ -1,8 +1,13 @@
+import { NotificationType } from './notification';
+
 // copied from /models/src/types/notification-category.req.d.ts
 export interface SaveNotificationCategoryRequest {
   name: string;
   description?: string;
-  notificationIds: string[];
+  /**
+   * @maxItems 2
```
Comment (mok0230) [inline packages/sentinel/src/models/category.ts:8]: why max 2 out of curiosity?

**Review by mok0230 — APPROVED** [review]
Looks good! Just a question

**Review by pasevin — APPROVED** [review]

File: packages/sentinel/src/models/category.ts (hunk 2 above)
Comment (shahnami) [reply packages/sentinel/src/models/category.ts:8]: Defined in the v0 scope. I'll DM you the slack conversation

**Pull request merged**